frontend/src/components/App/Home/index.tsx

```diff
@@ -20,18 +20,22 @@ import Event from '../../../lib/k8s/event';
 import { createRouteURL } from '../../../lib/router';
 import { useId } from '../../../lib/util';
 import { setConfig } from '../../../redux/configSlice';
+import { useTypedSelector } from '../../../redux/reducers/reducers';
 import { Link, PageGrid, SectionBox, SectionFilterHeader } from '../../common';
 import { ConfirmDialog } from '../../common';
+import ErrorBoundary from '../../common/ErrorBoundary/ErrorBoundary';
 import ResourceTable from '../../common/Resource/ResourceTable';
 import RecentClusters from './RecentClusters';
 
-function ContextMenu({ cluster }: { cluster: Cluster }) {
+function ContextMenu({ cluster }: { cluster: Cluster; active: boolean }) {
   const { t } = useTranslation(['translation']);
   const history = useHistory();
   const dispatch = useDispatch();
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
   const menuId = useId('context-menu');
-  const [openConfirmDialog, setOpenConfirmDialog] = React.useState(false);
+  const [openConfirmDialog, setOpenConfirmDialog] = React.useState('');
```

Comment:
please use undefined or null for empty value

Reply:
done